web/source/text/defaultOutput.ts

```diff
@@ -91,18 +141,26 @@ namespace com.keyman.text {
         //   if(code == VisualKeyboard.keyCodes['K_RIGHT']) {
         //     break;
         //   }
-        // // Should we include this?  It could be tricky to do correctly...
+      // }
+      //
+      // Note that these would be useful even outside of a DOM context.
+    }
+
+    /**
+     * Codes matched here may need special handling to apply to an OutputTarget, but still produce output.
```

Comment:
Can you explain the purpose of this function further because I'm a little uncertain even with this comment. When would this function be called, and how would its output be used? We wouldn't be injecting a `\b` into a text string for example?

This function has no default return value.